tasks/section5.yml

```diff
@@ -331,15 +465,19 @@
       - rule_5.2.9
 
 - name: "SCORED | 5.2.9 | PATCH | Ensure SSH PermitEmptyPasswords is disabled"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/ssh/sshd_config
+      regexp: '^PermitEmptyPasswords'
+      line: 'PermitEmptyPasswords no'
   tags:
       - level1
       - level2
       - patch
       - rule_5.2.9
 
 - name: "SCORED | 5.2.10 | AUDIT | Ensure SSH PermitUserEnvironment is disabled"
-  command: /bin/true
+  command: grep "^PermitUserEnvironment no" /etc/ssh/sshd_config
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*